server.js

```diff
@@ -27,21 +27,27 @@ app.prepare().then(() => {
     ) {
       // Enforce https protocol and remove www from host/
       res.writeHead(301, {
-        Location:
-          'https://' + req.headers['host'].replace(/^www\./, '') + req.url
+        Location: 'https://' + req.headers.host.replace(/^www\./, '') + req.url
       })
       res.end()
     } else if (req.headers.host === 'man.dvc.org') {
       // man.dvc.org/{cmd} -> dvc.org/doc/command-reference/{cmd},
       // replace - for / in {cmd} except for /get-url, /import-url
-      let new_path
-      if (['/get-url', '/import-url'].indexOf(pathname) < 0) {
-        new_path = pathname.replace('-', '/')
-      } else {
-        new_path = pathname
-      }
       res.writeHead(301, {
-        Location: 'https://dvc.org/doc/command-reference' + new_path
+        Location:
+          'https://dvc.org/doc/command-reference' +
+          (['/get-url', '/import-url'].indexOf(pathname) < 0
+            ? pathname.replace('-', '/')
+            : pathname)
+      })
+      res.end()
+    } else if (req.headers.host === 'man.localhost:3000') {
+      res.writeHead(301, {
+        Location:
+          'http://localhost:3000/doc/command-reference' +
+          (['/get-url', '/import-url'].indexOf(pathname) < 0
+            ? pathname.replace('-', '/')
+            : pathname)
       })
```

Comment:
In fact I added this code temporarily! Please download branch and run `yarn dev` and verify that:

http://man.localhost:3000/get-url -> http://localhost:3000/doc/command-reference/get-url
http://man.localhost:3000/import-url -> http://localhost:3000/doc/command-reference/import-url
http://man.localhost:3000/cache-dir -> http://localhost:3000/doc/command-reference/cache/dir
http://man.localhost:3000/no-thing -> http://localhost:3000/doc/command-reference/no/thing (not found)

Reply:
> Actually http://man.localhost:3000/import-url sends me to **https**://localhost:3000/doc/command-reference/import-url right now, Idk why. This causes a `ERR_SSL_PROTOCOL_ERROR` on Chrome. Whatever this is it wouldn't be a problem on prod which does support HTTPs.

Reply:
@jorgeorpinel please, see the last comment in the ticket.

Reply:
https://github.com/iterative/dvc.org/issues/768#issuecomment-549456704

^ Thanks for figuring the caching issue out. I was going crazy over here haha.

OK I'm reverting this commit with the man.localhost... redirect.